src/inmanta/db/util.py

```diff
@@ -52,8 +53,13 @@ async def __anext__(self) -> bytes:
 class PGRestore:
     """
     Class that offers support to restore a database dump.
+
+    This class assumes that the names of schemas, tables and columns in the dump don't contain a dot, double quote or
```

Comment:
I think this is a fair assumption to make, because we have full control over this. Do you agree?

Reply:
This sound reasonable. Just to make sure, when you say we have full control over this, you mean these names are the ones like `public.environment` etc which we will always name with letters only ?

Reply:
Yes. We create all these tables ourselves and we decide ourselves what the names of the tables and their columns are. We will never be crazy enough to put these weird characters is the name, so we are on the safe side.